x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiFieldSearch,
+  EuiSpacer,
+  EuiAccordion,
+  EuiPanel,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { FIELD_FILTER_CUTOFF } from '../constants';
+import { RelevanceTuningLogic } from '../relevance_tuning_logic';
+
+import { RelevanceTuningItem } from './relevance_tuning_item';
+import { RelevanceTuningItemContent } from './relevance_tuning_item_content';
+
+import './relevance_tuning_form.scss';
+
+export const RelevanceTuningForm: React.FC = () => {
+  const {
+    filterInputValue,
+    schemaFields,
+    filteredSchemaFields,
+    schema,
+    searchSettings,
+  } = useValues(RelevanceTuningLogic);
+  const { setFilterValue } = useActions(RelevanceTuningLogic);
+
+  return (
+    <section className="relevanceTuningForm">
+      <form>
+        {/* TODO SchemaConflictCallout */}
```

Comment:
OH the TODO is here. Hmmmm. It kinda makes sense to put it in the topmost view/layout though, next to flash message... am I crazy on that?

Reply:
Ah, yes. Most likely. I'll make sure when I implement it that it ends up in the most sensible place.